binderhub/app.py

```diff
@@ -234,6 +234,15 @@ def _add_slash(self, proposal):
         config=True
     )
 
+    build_node_selector = Dict(
+        default_value=None,
+        allow_none=True,
```

Comment:
rather than default=None, allow_none, we can leave the default as an empty dict, right?

Reply:
right. i am going to change it.